src/pages/index.tsx

```diff
@@ -1,231 +1,229 @@
 import { AnimatedLogo } from '@/components/animated-logo';
 import { Container } from '@/components/container';
 import { SiteFooter } from '@/components/site-footer';
-import { SiteHeader } from '@/components/site-header';
 
 export default function Home() {
-  return (
-    <main className="flex w-full min-h-screen flex-col items-center justify-between">
-      <SiteHeader />
-      <Container>
-        <div className="flex flex-col gap-3">
-          <AnimatedLogo />
+	return (
+		<main className="flex w-full min-h-screen flex-col items-center justify-between">
+			<Container>
```

Comment:
All I did here was remove the SiteHeader, as we are sticking with one page for now.

Reply:
Hehe, I'm actually in the process of putting in a news page so will need that header back 😅